clusters/coreos/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+pushd $( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+godep go build -a 
+
+docker build -t vish/heapster-buddy-coreos:canary .
```

Comment:
just "heapster-buddy-coreos" which keeps it local. If you want to push it to a repo you can do:

```
$ docker tag heapster-buddy-coreos kubernetes/heapster-buddy-coreos
$ docker push kubernetes/heapster-buddy-coreos
```